classes/Provider/SwiftMailerServiceProvider.php

```diff
@@ -25,12 +27,16 @@
      */
     public function register(Container $app)
     {
-        $app['mailer'] = function ($app) {
-            $transport = (new Swift_SmtpTransport($app->config('mail.host'), $app->config('mail.port')))
-                ->setUsername($app->config('mail.username'))
-                ->setPassword($app->config('mail.password'));
+        $app['mailer.transport'] = function ($app) {
+            return $app['env'] === Environment::TYPE_TESTING
```

Comment:
To enhance legibility, can we turn this ternary statement into an `if` with an early `return` instead?